src/client/components/description/Description.js

```diff
@@ -1,22 +1,50 @@
 import React from 'react';
-import headerPicture from '@root/client/images/headerPicture.jpg'
-import Head from './head/head';
-import Container from './container/Container';
-import styles from './Description.module.css';
-import cn from 'classnames';
+import Head from './head/Head'
```

Comment:
@AndreiBike please reorganize imports in such way that firstly should be imports from absolute path, then import from relative path

Reply:
like here
`
import { getMovieById } from "@root/client/store/store"
import Preloader from '@root/client/components/shared/preloader/Preloader';
import Head from './head/Head'
import Specification from './specification/Specification'
import './Description.module.css'

`

Reply:
Done)